multinet/multinet/graph.py

```diff
@@ -0,0 +1,31 @@
+from . import db
+from .types import *
+
+# a graph knows about the types of things in it and a set of nodes and edges
+
+# each edge type is a set of keys that can be pulled from various tables
+# that represent the possible properties of that edge
+def edge_types(graph, info):
+    return db.graph_edge_types(graph)
+
+# each node type is a set of keys that can be pulled from various tables
+# that represent the possible properties of that node
+def node_types(graph, info):
+    return db.graph_node_types(graph)
+
+def graph_name(graph, info):
```

Comment:
I think this should just be `name` since the method name can be fully qualified at the call site as `graph.name()`. What do you think?

Reply:
I had done similar things in other places and have now done it here too. I also simplified query and mutation.